config/olm/kubernetes/0.1.0/bundle/metadata/annotations.yaml

```diff
@@ -8,5 +8,3 @@ annotations:
   operators.operatorframework.io.metrics.builder: operator-sdk-v1.3.0
```

Comment:
the latest operator sdk is 1.13.0, please double check that this value is correct